cargo-dist/src/build/cargo.rs

```diff
@@ -156,34 +169,68 @@ impl<'a> DistGraphBuilder<'a> {
     }
 }
 
-// This function was split out of build_cargo_target() so it can have unit
-// tests on its own.
-fn make_build_cargo_target_command(
-    cargo_cmd: &String,
-    rustflags: &String,
-    target: &CargoBuildStep,
+/// Generate a `cargo build` command
+pub fn make_build_cargo_target_command(
+    cargo_cmd: &str,
+    rustflags: &str,
+    step: &CargoBuildStep,
     auditable: bool,
 ) -> Cmd {
-    let mut command = Cmd::new(cargo_cmd, "build your app with Cargo");
+    let host = cargo_dist_schema::target_lexicon::HOST;
```

Comment:
Note for myself: we can replace the `current_platform` usage with this once this PR is merged.

Reply:
So this is delicate — `target_lexicon::HOST` is "what the currently-running dist binary was built for".

On the same Linux system, you can get either `x86_64-unknown-linux-gnu` or `x86_64-unknown-linux-musl` depending on which dist binary you're running. That was surprising to me when the codepath suddenly started considering this a cross 🙃